src/platform-includes/enriching-events/breadcrumbs/automatic-breadcrumbs/apple.mdx

```diff
@@ -0,0 +1,5 @@
+The Cocoa SDK captures [breadcrumbs automatically](/platforms/apple/enriching-events/breadcrumbs/#automatic-breadcrumbs) for
```

Comment:
The link is linking back to this section; doesn't seem like it needs to be there
```suggestion
The Cocoa SDK captures breadcrumbs automatically for:
```